src/components/AddBook.js

```diff
@@ -1,18 +1,38 @@
-const AddBook = () => (
-  <div>
-    <form id="form-add">
-      <input type="text" name="title" placeholder="title" />
-      <input type="text" name="author" placeholder="author" />
+import { useDispatch } from 'react-redux';
+// import your Action Creators
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
+
+const AddBook = () => {
+  const dispatch = useDispatch();
+  const submitBookToStore = () => {
+    const parent = document.getElementById('form-add');
+    const newBook = {
+      id: uuidv4(), // make sure it's unique
+      title: parent.title.value,
+      author: parent.author.value,
+      genre: parent.genre.value,
+      progress: 30,
+    };
```

Comment:
- [ ] As it is part of the requirements for this project, being able to track changes in your inputs and update the local state, please kindly set a local state for your component with `title` and `author`, and the corresponding inputs should update that local state every time they update(`onChange`). See screenshot of the requirements below for clarifications. 
![localState](https://user-images.githubusercontent.com/65852621/150111892-7d711569-24b2-4ba7-8b26-4d67b01d3b6b.png)

Reply:
@Mihndim2020 The above requirement does not exist in the project requirements section of this project. Rather this is how the requirement looks and I have fulfilled it. https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_create_and_remove_book.md